tuxemon/ai.py

```diff
@@ -81,12 +95,49 @@ def make_decision(
 class RandomAI(AI):
     """AI will use random technique against random opponent."""
 
-    def make_decision(
+    def make_decision_trainer(
+        self,
+        user: NPC,
+        monster: Monster,
+        opponents: Sequence[Monster],
+    ) -> Tuple[Union[Monster, NPC], Union[Item, Technique], Monster]:
+        """
+        Trainers battles.
+        """
+        if self.check_strongest(user, monster):
+            if len(user.inventory) > 0:
+                inventory = list(user.inventory)

```

Comment:
what is the type of the inventory? Do we actually need to cast it into a list?

Reply:
inventory <class 'dict'>
Cast into a list because I thought it was more handy than loop in the dict.